apps/style/curriculum/reference_guides.scss

```diff
@@ -0,0 +1,29 @@
+@import 'color';
```

Comment:
For my knowledge, why use this set up instead of putting the styles in the react components?

Reply:
This is closer to what we're looking at for the scss+modules proposal here: https://docs.google.com/document/d/1Y3uK_iYMhTUaCI6yIDwOAMprIJCuzXSs2fECQggwp60/edit#heading=h.htf3pg55q2kt
It'd be less work to migrate from already having an scss file (and in my opinion it is easier to organize styles using css selectors vs style objects in js)